src/main/java/org/jenkinsci/plugins/GithubRequireOrganizationMembershipACL.java

```diff
@@ -239,6 +250,14 @@ private boolean checkReadPermission(@NonNull Permission permission) {
                 || id.equals("hudson.model.Item.Read"));
     }
 
+    private boolean checkAgentUserPermission(@NonNull Permission permission) {
+        String id = permission.getId();
+        return (id.equals("hudson.model.Hudson.Read")
```

Comment:
I didn't find that `Read` was required in my testing. Was this something you had to include to resolve an issue? Else I think we should remove it (principle of least privilege).

It is also not covered in the unit tests, and so we'd need to update them if we need to keep this.

Reply:
I was following the documentation of the docker-swarm-plugin in which it is mentioned that the general "Read" right is required: https://github.com/jenkinsci/swarm-plugin/blob/master/docs/security.adoc#authorization.
Not sure why it should be needed as I agree with you that we should go with the principle of least privilege.